include/zephyr/lorawan/lorawan.h

```diff
@@ -295,6 +311,20 @@ int lorawan_set_conf_msg_tries(uint8_t tries);
  */
 void lorawan_enable_adr(bool enable);
 
+/**
+ * @brief Set the channels mask.
+ *
+ * Change the default channels mask. When mask is not changed, all the channels
+ * can be used for data transmition. Some Network Servers don't use all channels,
+ * in this case, the channels mask must be provided.
+ *
+ * @param channels_mask Buffer with channels mask to be used.
+ * @param channels_mask_size Size of channels mask buffer.
+ *
+ * @return 0 if successful, negative errno code if failure
```

Comment:
Add the @ret val for each return code. 
```suggestion
 * @retval 0 successful
 * @retval -EXXXX <justify>
```

Reply:
Fixed.